src/components/subscriptions/data/service.js

```diff
@@ -130,16 +150,11 @@ export function fetchSubscriptionUsers(options = {}) {
  * call to get this data.
  */
 export function sendLicenseReminder(options = {}) {
-  return Promise.resolve(options);
-}
+  const { userId, bulkRemind } = options;
+  const pendingSince = moment().format('MMM DD, YYYY');
+  const response = updateUserRemindTimeStamp({ userId, bulkRemind, pendingSince });
 
-/**
- * This function mocks out the response from a non-existant API endpoint. Once the endpoint
- * exists, the contents of this function will use the `apiClient` to make an actual API
- * call to get this data.
- */
-export function addLicensesForUsers(payload) {
```

Comment:
Are you intentionally removing this `addLicensesForUsers` function? Getting an error about it when running `npm run start`:

<img width="962" alt="image" src="https://user-images.githubusercontent.com/2828721/85729244-a23e6780-b6c6-11ea-82be-6d80b69c2745.png">

Reply:
I think I have mistakenly removed it while resolving conflicts.